index.html

```diff
@@ -163,6 +163,91 @@ <h1 class="clr-blue">About me</h1>
       </p>
       <button class="btn btn-2">Get my resume</button>
     </section>
-    <section></section>
+    <section class="skills center">
+      <div class="skill">
+        <img src="imgs/p1.png" alt="skills" class="p1" />
+        <h1>Languages</h1>
+        <ul class="li">
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">HTML</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">CSS</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Javascript</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Ruby</li>
+          &nbsp;&nbsp;&nbsp;
+        </ul>
+      </div>
+    </section>
+    <section class="skills center frameworks">
+      <div class="skill">
+        <img src="imgs/p2.png" alt="skills" class="p1" />
+        <h1>Frameworks</h1>
+        <ul class="li li-3">
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">React.js</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Ruby</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">RSpec</li>
+          &nbsp;&nbsp;&nbsp;
+        </ul>
+        <ul class="li li-3">
+          <li class="li-2">Capybara</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Selenium</li>
+          &nbsp;&nbsp;&nbsp;
+        </ul>
+      </div>
+    </section>
+    <section class="skills center">
+      <div class="skill">
+        <img src="imgs/p3.png" alt="skills" class="p1" />
+        <h1>Skills</h1>
+        <ul class="li">
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Database Management</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Version Control</li>
+          &nbsp;&nbsp;&nbsp;
+        </ul>
+        <ul class="li">
+          <li class="li-2">CLI</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">Web Development</li>
+          &nbsp;&nbsp;&nbsp;
+          <li class="li-2">API Design</li>
+          &nbsp;&nbsp;&nbsp;
+        </ul>
+      </div>
+    </section>
+    <section class="content-4 center">
+      <h1 class="clr-blue">
+        I'm always interested in hearing about new projects, so if you'd like to
+        chat please get in touch.
+      </h1>
+    </section>
+    <section class="center">
+      <input type="text" class="in" placeholder="&nbsp;&nbsp;&nbsp;Full name" />
+      <input
+        type="text"
+        class="in"
+        placeholder="&nbsp;&nbsp;&nbsp;Email address"
+      />
+      <textarea
+        name="text"
+        id=""
+        cols="30"
+        rows="10"
+        placeholder="
+        Hey I’d like to connect to..."
+      ></textarea>
+      <img class="foot" src="imgs/footer.png" alt="illustration" />
+      <br />
+      <button class="btn btn-2 btn-3">Get in touch</button>
+      <p class="clr-blue">blesslinjerish@gmail.com</p>
+      <img src="imgs/socialmedia.png" alt="social media" />
+    </section>
```

Comment:
- Kindly remove this section as it's not part of the requirement for this milestone. (You'll start working on the form in more detail next week 🏁)